apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -88,6 +88,18 @@ public function register_patterns() {
 
 			$pattern_categories = array_merge( $pattern_categories, $existing_categories );
 
+			// The category 'All' is created dinamically so all patterns are always added automatically.
+			$category_all = array(
```

Comment:
[nit] Possible to move this declaration outside the `foreach` loop? So that it's not declared every time 🙂